float32 normalDistanceWeight
int64 maxIterations
float32 distanceThreshold
---
